app/src/main/res/layout/item_view.xml

```diff
@@ -26,13 +25,12 @@
         android:layout_width="@dimen/match_constraint"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/standard_margin"
-        android:layout_marginLeft="@dimen/standard_margin"
         android:text="@string/description"
         android:textColor="@color/description_text_color"
         android:textSize="@dimen/text_size"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/title_text"
         app:layout_constraintVertical_chainStyle="packed" />
 
```

Comment:
This is how the layout looks, its side paddings are not equal:

![Screenshot_1575091348](https://user-images.githubusercontent.com/8268042/69895981-b8cd4c00-138d-11ea-8d49-1aad10c9a657.png)